tests/ui/target_modifiers/defaults_check.error.stderr

```diff
@@ -0,0 +1,13 @@
+error: mixing `-Zreg-struct-return` will cause an ABI mismatch
+  --> $DIR/defaults_check.rs:13:1
+   |
+LL | #![crate_type = "lib"]
+   | ^
+   |
```

Comment:
Discussion: this seems somewhat strange to point to the `crate_type` attribute. Is it possible to instead report the name of "this crate" instead?

Reply:
I have added crate name in error notification:
```
error: mixing `-Zregparm` will cause an ABI mismatch in crate `incompatible_regparm`
```
Span itself points to `krate.spans.inner_span.shrink_to_lo()`. I don't think that there may be a better span. Only to remove span completely, maybe. But then you will not be able to open problem crate in IDE by click, I think.

Reply:
Hm true. I think this is fine.